src/sources/opentelemetry/mod.rs

```diff
@@ -121,15 +137,102 @@ impl SourceConfig for OpentelemetryConfig {
         let http_tls_settings = MaybeTlsSettings::from_config(&self.http.tls, true)?;
         let protocol = http_tls_settings.http_protocol_name();
         let bytes_received = register!(BytesReceived::from(Protocol::from(protocol)));
-        let filters = build_warp_filter(acknowledgements, cx.out, bytes_received);
+        let filters = build_warp_filter(acknowledgements, log_namespace, cx.out, bytes_received);
         let http_source =
             run_http_server(self.http.address, http_tls_settings, filters, cx.shutdown);
 
         Ok(join(grpc_source, http_source).map(|_| Ok(())).boxed())
     }
 
-    fn outputs(&self, _global_log_namespace: LogNamespace) -> Vec<Output> {
-        vec![Output::default(DataType::Log).with_port(LOGS)]
+    fn outputs(&self, global_log_namespace: LogNamespace) -> Vec<Output> {
+        let log_namespace = global_log_namespace.merge(self.log_namespace);
+        let schema_definition = Definition::new_with_default_metadata(Kind::any(), [log_namespace])
+            .with_source_metadata(
+                Self::NAME,
+                Some(LegacyKey::Overwrite(owned_value_path!(RESOURCE_KEY))),
+                &owned_value_path!(RESOURCE_KEY),
+                Kind::object(Collection::from_unknown(Kind::any())).or_undefined(),
+                None,
+            )
+            .with_source_metadata(
+                Self::NAME,
+                Some(LegacyKey::Overwrite(owned_value_path!(ATTRIBUTES_KEY))),
+                &owned_value_path!(ATTRIBUTES_KEY),
+                Kind::object(Collection::from_unknown(Kind::any())).or_undefined(),
+                None,
+            )
+            .with_source_metadata(
+                Self::NAME,
+                Some(LegacyKey::Overwrite(owned_value_path!(TRACE_ID_KEY))),
+                &owned_value_path!(TRACE_ID_KEY),
+                Kind::bytes().or_undefined(),
+                None,
+            )
+            .with_source_metadata(
+                Self::NAME,
+                Some(LegacyKey::Overwrite(owned_value_path!(SPAN_ID_KEY))),
+                &owned_value_path!(SPAN_ID_KEY),
+                Kind::bytes().or_undefined(),
+                None,
+            )
+            .with_source_metadata(
+                Self::NAME,
+                Some(LegacyKey::Overwrite(owned_value_path!(SEVERITY_TEXT_KEY))),
+                &owned_value_path!(SEVERITY_TEXT_KEY),
+                Kind::bytes().or_undefined(),
+                None,
+            )
+            .with_source_metadata(
+                Self::NAME,
+                Some(LegacyKey::Overwrite(owned_value_path!(SEVERITY_NUMBER_KEY))),
+                &owned_value_path!(SEVERITY_NUMBER_KEY),
+                Kind::integer().or_undefined(),
+                None,
```

Comment:
```suggestion
                Some("severity"),
```

I think either this or the severity text should have the severity meaning.

Reply:
Resolved in [fb2a0bb](https://github.com/vectordotdev/vector/pull/15469/commits/fb2a0bb78117cda60ad07ba844122463931989c3)